tokens/models/tokens/fuse/tokens_fuse_erc20.sql

```diff
@@ -0,0 +1,18 @@
+{{
+    config(
+        schema = 'tokens_fuse'
+        ,alias = 'erc20'
+        ,tags = ['static']
+        ,materialized = 'table'
+    )
+}}
+
+SELECT
+    contract_address
+    , symbol
+    , decimals
+FROM (VALUES
+    (0x34ef2cc892a88415e9f02b91bfa9c91fc0be6bd4, 'VOLT', 18),
```

Comment:
https://explorer.fuse.io/token/0x34Ef2Cc892a88415e9f02b91BfA9c91fC0bE6bD4

Reply:
Do I have to check something?

Reply:
Ohhh okayy 😄